src/mongocxx/test/database.cpp

```diff
@@ -33,6 +34,18 @@ using bsoncxx::builder::stream::document;
 using bsoncxx::builder::stream::finalize;
 using bsoncxx::builder::stream::open_document;
 
+namespace {
+
+int get_max_wire_version(const client& mongo_client) {
```

Comment:
This seems like the kind of thing that should go into the helper header or into a private .cpp file.  This is likely to be useful elsewhere over time.